jupyter-extension/setup.cfg

```diff
@@ -19,7 +19,7 @@ classifiers =
 packages = find:
 install_requires =
   pyyaml>=6.0
-  jupyter_server>=1.6,<2
+  jupyter_server==2.7.0
```

Comment:
We should be able to loosen this pinning to `>=2.7.0,<2.8` to allow patches/bug-fixes to bubble up.

Reply:
Can do.   Determined is fixed to the single 2.7.0, but that doesn't mean Pachyderm has to be.
The jupyter_server recommendation was to pin to a specific version to avoid incompatibilities.   
Certainly patch versions are likely to be OK, but their example was to use and exact version (see below).

https://jupyter-server.readthedocs.io/en/latest/developers/dependency.html

> If your project depends directly on Jupyter Server, be sure to watch Jupyter Server’s ChangeLog and pin your project to a version that works for your application. Major releases represent possible backwards-compatibility breaking API changes or features.
```
To pin your jupyter_server install to a specific version:

> pip install jupyter_server==1.0.0

```

Reply:
Updated as suggested.